package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "test": "export NODE_ENV=test && mocha --recursive"
+    "test": "nyc mocha --recursive"
```

Comment:
```suggestion
    "test": "export NODE_ENV=test && nyc mocha --recursive"
```

Reply:
We still need to specify that the tests are running in our test database (not development environment)